lobbyboy/contrib/provider/linode.py

```diff
@@ -0,0 +1,134 @@
+import os
```

Comment:
Thanks for this! 👍

can you add this to `readme.md`? https://github.com/laixintao/lobbyboy#builtin-providers

Reply:
sure, my pleasure🥳